variables.tf

```
@@ -55,6 +55,12 @@ variable "bucket_force_destroy" {
   default = false
 }
 
+variable "bucket_region" {
+  type        = string
+  default     = "us"
+  description = "The region where the new bucket will be created, valid values (eu, us or asia)"
```

if so, then this message should be updated
```suggestion
  description = "The region where the new bucket will be created, valid values (EU, US or ASIA)"
```

---

@afiune my latest commit now reflects the changes